### ISSUE TITLE
Doesn't Unlock or lock darkrp doors

### ISSUE THREAD

**Issue by KyberNetworks** [MattJeanes/Sonic-Screwdriver#22]
It doesn't lock or unlock darkrp doors anymore like it used too

**Comment by KyberNetworks** [comment]
I have found out why its FPP tool protection stopping it working

**Comment by MattJeanes** [comment]
Yeah prop protection preventing the sonic is intended behaviour, but thanks for checking back on this issue

**Comment by KyberNetworks** [comment]
Sorry to reopen this but is there a code you can give me to bypass it so they can only use the sonic on blocked wolrd models in fpp as if i turn it off people will be using tools on world  doors

**Comment by MattJeanes** [comment]
That's okay, yeah I made it extensible so you can override behaviour like this.

So basically what you're gonna want to do is drop a file into lua/weapons/swep_sonicsd/modules (this can be in any addon or even directly into the gmod folder) called something like sv_dooroverride.lua

In this file you should have something like:

```lua
SWEP:AddHook("CanTool", "doctorwho", function(self, owner, ent)
    if ent:GetClass() == "prop_rotating_door" then
        return true
    end
end)
```

Of course you should change the logic as appropriate, this example will allow you to always lock/unlock doors regardless of prop protection settings. Hope this helps

**Comment by KyberNetworks** [comment]
does this still working as players cant unlock or lock doors anymore

**Comment by KyberNetworks** [comment]
admins can use it when they are able to use tool on block ents
